src/titan_db_test.cc

```diff
@@ -355,21 +355,8 @@ TEST_F(TitanDBTest, Open) {
         background_job_started = true;
         assert(false);
       });
-  SyncPoint::GetInstance()->SetCallBack(
```

Comment:
This test case does not have any special meaning, I guess. The new change breaks this, because by the time "BeforeOpenBlobFileSet", `default_cf_handle_` is not ready yet. We have other means to fix it, but the most graceful way is to remove it.